skyplane/obj_store/hdfs_interface.py

```diff
@@ -0,0 +1,49 @@
+import os
+from pyarrow import fs
+from dataclasses import dataclass
+from skyplane.obj_store.file_system_interface import FileSystemInterface, LocalFile
+from typing import Iterator, List, Optional, Tuple
+
+
+@dataclass
+class HDFSFile(LocalFile):
```

Comment:
Is there any reason to declare a new `HDFSFile` that is the same as the underlying `LocalFile` object? We can just use the `LocalFile` object perhaps?

Reply:
You need the new class since the methods are undefined in the LocalFile. Gotcha